### ISSUE TITLE
[UWP] ScaleTo appears to no longer work

### ISSUE THREAD

**Issue by PureWeen** [xamarin/Xamarin.Forms#3279]
### Description
ScaleTo doesn't appear to do anything anymore on UWP

### Steps to Reproduce
Using the following code click on the text. 

On iOS and Android you can see the scale change and then come back whereas on UWP it does nothing

```C#
Label TheLabel = new Label() { Text = "Welcome to Xamarin.Forms!" };
Content = TheLabel;
var tapGesture = new TapGestureRecognizer();
var animateCommand = new Command(async () =>
{
	await TheLabel.ScaleTo(.92, 100);
	await TheLabel.ScaleTo(1, 100);
});
tapGesture.Command = animateCommand;
TheLabel.GestureRecognizers.Add(tapGesture);
```

### Expected Behavior

The Label should shrink a little then grow back to normal size

### Actual Behavior
Clicking on the Text has no effect

### Basic Information

- Version with issue: 3.1+ (also tested on master and 3.2)
- Last known good version: 3.0

**Comment by deakjahn** [comment]
Scale itself doesn't work, although it was supposed to be fixed about two weeks ago. As Scale stays at 1, the ScaleTo() seems to optimize it away. I'll locate the ref to the previous issue.

**Comment by deakjahn** [comment]
Was this supposed to be fixed in the new 3.1.697729? It still isn't working.

**Comment by paymicro** [comment]
The scale works correctly on the master branch.
![](http://g.recordit.co/5UnOSK4xIL.gif)
Looks it's already been fixed by PR https://github.com/xamarin/Xamarin.Forms/pull/3178.

**Comment by rmarinho** [comment]
@paymicro  is right,. This should be fixed on 3.3.0-pre1
